match/lib/match/git_helper.rb

```diff
@@ -57,7 +59,7 @@ def self.commit_changes(path, message, git_url)
         commands = []
         commands << "git add -A"
         commands << "git commit -m '#{message}'"
```

Comment:
While it is not a part of your change, I wanted to note that this line is also a shell escaping problem for any message containing a single quote. While it makes the generated command text very ugly, unquoted `shellescape` is the only way to be safe:

```ruby
commands << "git commit -m #{message.shellescape}"
```